library/unwind/src/lib.rs

```diff
@@ -55,6 +55,24 @@ cfg_if::cfg_if! {
     }
 }
 
+#[cfg(target_os = "android")]
+cfg_if::cfg_if! {
+    if #[cfg(feature = "system-llvm-libunwind")] {
+        #[link(name = "unwind", kind = "static", modifiers = "-bundle", cfg(target_feature = "crt-static"))]
+        #[link(name = "unwind", cfg(not(target_feature = "crt-static")))]
+        extern "C" {}
+    } else {
+        #[link(name = "gcc", kind = "static", modifiers = "-bundle", cfg(target_feature = "crt-static"))]
+        #[link(name = "gcc", cfg(not(target_feature = "crt-static")))]
+        extern "C" {}
+    }
+}
```

Comment:
Could you add a `compile_error!(...)` for impossible combinations of llvm-libunwind features, similarly to `#[cfg(target_env = "musl")]` above?

Reply:
`feature = "llvm-libunwind"` by itself is also impossible for Android.
(Also `} else if` is formatted on one line.)